test-kotest5/src/main/kotlin/io/micronaut/test/extensions/kotest5/MicronautKotest5Extension.kt

```diff
@@ -51,6 +51,7 @@ object MicronautKotest5Extension: TestListener, ConstructorExtension, TestCaseEx
 
     override suspend fun afterSpec(spec: Spec) {
         contexts[spec.javaClass.name]?.afterSpecClass(spec)
+        contexts[spec.javaClass.name]?.close()
```

Comment:
The line above already closes the application context:

https://github.com/micronaut-projects/micronaut-test/blob/master/test-core/src/main/java/io/micronaut/test/extensions/AbstractMicronautExtension.java#L455